util/math_reference.cpp

```diff
@@ -336,6 +345,7 @@ T mul_hi_signed(T a, T b) {
   r = static_cast<U>(a_pos) & mask;
   s = static_cast<U>(b_pos) & mask;
 
+  // calc half products
```

Comment:
Comments are expected to be read by human beings
```suggestion
  // Compute half products
```

Reply:
Fixed. Thanks.